tests/conftest.py

```diff
@@ -222,17 +222,24 @@ def _framework_fixture_inner(request, app, slug, **kwargs):
 
 
 def _supplierframework_fixture_inner(request, app, sf_kwargs=None):
+    """
```

Comment:
Really? I don't like it. I'm just cramming it in there until someone factory boys our api tests...